src/plugins/discover/public/application/angular/doc_table/components/_table_header.scss

```diff
@@ -1,3 +1,6 @@
+.kbnDocTableHeader th{
```

Comment:
```suggestion
.kbnDocTableHeader {
```

This is safe against the whole header. You don't want to use this against a naked dom element.

Reply:
thx @snide !